docs/src/main/asciidoc/security-keycloak-authorization.adoc

```diff
@@ -267,18 +278,21 @@ quarkus.keycloak.devservices.realm-path=quarkus-realm.json <6>
 <1> Specifies the URL of the Keycloak server and the realm used for authentication.
 <2> Identifies the client application within the Keycloak realm.
 <3> Defines the client secret for authentication with the Keycloak server.
-<4> Disables TLS verification for development purposes. Not recommended for production.
+<4> Disables TLS verification for development purposes.
+Not recommended for production.
```

Comment:
May be it should be a single sentence connected with `,` ?

Reply:
Updated. Thanks.